hack/update-codegen.sh

```diff
@@ -762,6 +762,43 @@ function codegen::subprojects() {
     done
 }
 
+function codegen::protobindings() {
+    # Each element of this array is a directory containing subdirectories which
+    # eventually contain a file named "api.proto".
+    local apis=(
```

Comment:
does anything ensure each directory in this list exists and contains an api.proto file somewhere, or does a typo or incorrect directory silently get ignored?

Reply:
new commit added